idseq_dag/steps/run_bowtie2.py

```diff
@@ -47,4 +48,4 @@ def run(self):
                 output_sam_file, output_fas[0])
 
     def count_reads(self):
-        self.count_reads_work()
+        self.counts_dict[self.name] = count.reads_in_group(self.output_files_local()[0:2])
```

Comment:
I know is just one line, but I wonder if it is worth to create a parent class that does this to avoid copy paste?